src/rt/sched/behaviourcore.h

```diff
@@ -190,16 +457,22 @@ namespace verona::rt
      *
      * Returns true if this call makes the count_down_zero
      */
-    void resolve(size_t n = 1)
+    bool resolve(size_t n = 1, bool fifo = true)
```

Comment:
No one appears to be be using the return value.
```suggestion
     */
    void resolve(size_t n = 1, bool fifo = true)
```